### ISSUE TITLE
Google style indentation

### ISSUE THREAD

**Issue by thernstig** [gajus/eslint-plugin-jsdoc#715]
## Motivation

Indentation matters for readability, especially having a consistent indentation of documentation. I want to be able to use google style indentation for `@parameter` and `@return` as an example.

## Current behavior

This might be part of `check-indentation` but I could not get it work with it.

## Desired behavior

I want this to be invalid:
```js
  /**
   * Do something.
   *
   * @param {string} fooA - Some text
   * @param {string} fooB - If defined, it foos
   * a lot all day long.
   */
```

I want this to be valid (with the new rule)
```js
  /**
   * Do something.
   *
   * @param {string} fooA - Some text
   * @param {string} fooB - If defined, it foos
   *     a lot all day long. // This has been double-indented (4 spaces, we use 2 spaces for indents)
   */
```

This is called google style indent. It makes it easier to read the documentation.

**Comment by brettz9** [comment]
I don't think it would be too hard, especially on its own, but I think we'd want to integrate it into `check-line-alignment`, so that there would not be inconsistencies. We could avoid forcing users to use that rule's never or always options, while still enforcing a specific indent. Haven't been feeling too well, so not sure about implementing, unfortunately...

**Comment by thernstig** [comment]
@brettz9 ok I hope you are feeling ok soon!

**Comment by brettz9** [comment]
@thernstig : Would you mind taking #964 for a spin to see if it works well for you?
